### PR TITLE
upper camel case

### PR DESCRIPTION
**Pull request by christopher-cudennec**

### CODE REVIEW

**Review by JoergSiebahn — APPROVED** [review]
I leave it up to you, to include a „BREAKING CHANGE“ commit message on merge.

File: sda-commons-server-jackson/src/test/java/org/sdase/commons/server/jackson/errors/JerseyValidationExceptionMapperTest.java [hunk 1]
```diff
@@ -17,43 +18,43 @@ class JerseyValidationExceptionMapperTest {
...
 
+  /** Custom test data for edge case / custom testing. */
   static Stream<Arguments> testData() {
     return Stream.of(
         Arguments.of("NotNull", "NOT_NULL"),
         Arguments.of("Email", "EMAIL"),
-        Arguments.of("ISBN", "I_S_B_N"), // how Guava did it, default Jackson: "ISBN"
-        Arguments.of("ValidURI", "VALID_U_R_I"), // how Guava did it, default Jackson: "VALID_URI"
+        Arguments.of("ISBN", "ISBN"),
+        Arguments.of("ValidURI", "VALID_URI"),
```
[the remark this answers is not in the document]
Comment (christopher-cudennec) [reply sda-commons-server-jackson/src/test/java/org/sdase/commons/server/jackson/errors/JerseyValidationExceptionMapperTest.java]: I added a breaking change description in the [commit](https://github.com/SDA-SE/sda-dropwizard-commons/pull/1920/commits/3d202ee756dc8e64099c4b7b6cad2ee10f70a10b).
